test/max-requests.js

```diff
@@ -0,0 +1,118 @@
+const del = require('del');
+const mongo = require('emulate-mongo-2-driver');
+const { expect } = require('chai');
+const rp = require('request-promise');
+const enableDestroy = require('server-destroy');
+const apostropheMultisite = require('./app.js');
+const Promise = require('bluebird');
+
+describe('Apostrophe-multisite', function() {
+  describe('#dashboard', function() {
+    const port = 3000;
+    const admin = 'admin';
+    const url = 'site.test';
+    const title = 'New Site';
+    const shortNamePrefix = 'test-multi-';
+
+    let multisite;
+    let sites;
+    let site;
+    let req;
+    let exited;
```

Comment:
I did not catch why this needs to be declared here instead of inside the `exit()` function (and why the `exit()` function is needed in the test suite)

Reply:
I might be missing something, but I also don't see where this is used. I'm guessing you used it during development, Tom?

Reply:
It's passed into apostrophe-multisite so we don't actually exit the process (and break the tests) but just note whether we would have. However I wasn't actually using expect to verify that flag got set. Now I am.